src/Workleap.OpenApi.MSBuild/HttpClientWrapper.cs

```diff
@@ -6,6 +6,11 @@ internal sealed class HttpClientWrapper : IHttpClientWrapper, IDisposable
 
     public async Task DownloadFileToDestinationAsync(string url, string destination, CancellationToken cancellationToken)
     {
+        if (File.Exists(destination))
```

Comment:
Line 27 there's an exception thrown mentioning that Spectral could not be downloaded, but we use this to also download Oasdiff. The message should be changed